client/src/components/Billing/Checkout.js

```diff
@@ -2,10 +2,15 @@ import React from 'react'
 import axios from 'axios';
 import StripeCheckout from 'react-stripe-checkout';
 
-const PAYMENT_SERVER_URL = process.env.REACT_APP_PAYMENT_SERVER_URL;

```

Comment:
I commented out these lines. So the lines further down with the ternary operator would be the only PAYMENT_SERVER_URL lines.